src/IconButtonToggle/README.md

```diff
@@ -12,12 +12,10 @@ notes: |
   Helps toggle Data view between list, card view etc.
 ---
 
-<p>
```

Comment:
Removed `<p>` wrapper since it makes mdx render backticks as raw text instead of applying styles to words inside it.

Reply:
Ha, thanks for fixing.